README.md

```diff
@@ -64,17 +64,16 @@ Providing one's own version of [the images automatically built from this reposit
 kubectl create -f es-discovery-svc.yaml
 kubectl create -f es-svc.yaml
 kubectl create -f es-master.yaml
-```
-
-Wait until `es-master` deployment is provisioned, and
-```
+kubectl rollout status -f es-master.yaml
 kubectl create -f es-client.yaml
 kubectl create -f es-data.yaml
```

Comment:
Have the `rollout status` run first, so that:
```
kubectl create -f es-data.yaml
kubectl rollout status -f es-data.yaml
kubectl create -f es-client.yaml
kubectl rollout status -f es-client.yaml
```

Reply:
I'll squash - note that GitHub allows to do that during merge (the second option).

Reply:
Yeah, I know it does, but if you pgp sign your commits like I do, squash remove the signature and that's why I asked you. I can do it still if you don't.